uportal-war/src/main/resources/layout/theme/respondr/content.xsl

```diff
@@ -151,6 +160,15 @@
       </xsl:choose>
     </xsl:variable>
 
+    <!-- Applied in the case of DLM fragment owners only;  tells the UI to
+         permit the fragment admin to manage content even when it it locked. -->
+    <xsl:variable name="FRAGMENT_OWNER_CSS">
+      <xsl:choose>
+        <xsl:when test="$IS_FRAGMENT_ADMIN_MODE='true'">up-fragment-admin</xsl:when>
+        <xsl:otherwise></xsl:otherwise>
+      </xsl:choose>
+    </xsl:variable>
+
```

Comment:
Rather than duplicating the variable and conditional check, I suggest defining it once in respondr.xsl near the other fragment admin variable (IS_FRAGMENT_ADMIN_MODE) so it can be used anywhere.